### PR TITLE
VACMS-4347: Add default details class to required fieldsets on clp.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
New details with image formatter doesn't add the asterisk to fieldsets with required fields.
This pr adds the missing asterisk hook class to the fieldsets that are required.

## Testing done
Visual

## Screenshots
image

## QA steps
As an administrator, go to `/node/add/campaign_landing_page` and visually verify that the first three and the last fieldsets show a required red asterisk.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/core.entity_form_display.node.campaign_landing_page.default.yml [hunk 1]
```diff
@@ -98,11 +98,11 @@ third_party_settings:
       format_settings:
         description: 'The hero banner attracts attention, announces the campaign''s main message, and adds visual interest.'
         open: '1'
-        visual_guide_file_name: hero-banner-guide.svg
-        visual_guide_alt_text: 'Basic visual layout of the hero banner page segment.'
```
Comment (kevwalsh) [inline config/sync/core.entity_form_display.node.campaign_landing_page.default.yml:102]: i don't think you meant to do this.

**Review by kevwalsh — CHANGES_REQUESTED** [review]
I see the asterisk but the first image is missing.

File: config/sync/core.entity_form_display.node.campaign_landing_page.default.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.node.campaign_landing_page.default.yml:102]: Thanks for catching that, @kevwalsh :heart:

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image
  Tests out good for me.  Can merge when tests pass.

**Comment by swirtSJW** [comment]
image

**Pull request merged**